front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorFormHelpers.tsx

```diff
@@ -247,44 +282,131 @@ export const RollingStockEditorParameterForm = ({
   setOptionValue,
   setRollingStockValues,
   effortCurves,
+  rollingStockSchemasProperties,
 }: RollingStockEditorParameterFormProps) => {
   const { t } = useTranslation(['rollingstock', 'translation']);
+  const {
+    left: leftSideList,
+    middle: middleSideList,
+    right: rightSideList,
+  } = groupBy(rollingStockSchemasProperties, 'side') as { [key: string]: SchemaProperty[] };
   return (
-    <div className="d-lg-flex justify-content-center px-1 pb-3">
-      <div className="col-lg-4 rollingstock-editor-input-container">
+    <div className="d-xl-flex justify-content-center px-1 pb-3">
+      <div className="col-xl-4 rollingstock-editor-input-container mb-3">
         <RollingStockEditorParameterFormColumn
           optionValue={optionValue}
-          formSide="left"
           rollingStockValues={rollingStockValues}
           setOptionValue={setOptionValue}
           setRollingStockValues={setRollingStockValues}
           effortCurves={effortCurves}
+          propertiesList={leftSideList}
+          rollingStockSchemasProperties={rollingStockSchemasProperties}
         />
       </div>
-      <div className="col-lg-4 rollingstock-editor-input-container">
+      <div className="col-xl-4 rollingstock-editor-input-container mb-3">
         <RollingStockEditorParameterFormColumn
           optionValue={optionValue}
-          formSide="middle"
           rollingStockValues={rollingStockValues}
           setOptionValue={setOptionValue}
           setRollingStockValues={setRollingStockValues}
           effortCurves={effortCurves}
+          propertiesList={middleSideList}
+          rollingStockSchemasProperties={rollingStockSchemasProperties}
         />
       </div>
-      <div className="col-lg-4">
-        <div className="d-flex flex-column mb-2">
-          <span className=" ml-2 text-gray-dark">{t('rollingResistance')}</span>
+      <div className="d-flex flex-column justify-content-between col-xl-4 pb-3">
+        <div className="d-flex flex-xl-column mb-2 mt-3 mt-xl-0">
+          <span className=" ml-xl-2 text-gray-dark">{t('rollingResistance')}</span>
           <span className=" ml-4 text-muted">{t('rollingResistanceFormula')}</span>
         </div>
         <RollingStockEditorParameterFormColumn
           optionValue={optionValue}
-          formSide="right"
           rollingStockValues={rollingStockValues}
           setOptionValue={setOptionValue}
           setRollingStockValues={setRollingStockValues}
           effortCurves={effortCurves}
+          propertiesList={rightSideList}
+          rollingStockSchemasProperties={rollingStockSchemasProperties}
         />
       </div>
     </div>
   );
 };
+
+type RollingStockEditorOnboardSystemEquipmentFormProps = {
+  rollingStockValues: RollingStockParametersValues;
+  setRollingStockValues: (
+    rollingStockValue: React.SetStateAction<RollingStockParametersValues>
+  ) => void;
+  rollingStockSchemasProperties: readonly SchemaProperty[];
+};
```

Comment:
Simplify the props like this. Don't try to make a generate component, we don't have this need for the moment and it's complicating the code for nothing :) 
```suggestion
type RollingStockEditorOnboardSystemEquipmentFormProps = {
  rollingStockSignalingSystems: string[];
  setRollingStockValues: (
    rollingStockValue: React.SetStateAction<RollingStockParametersValues>
  ) => void;
};
```